packages/admin-ui-react/src/createSystem.tsx

```diff
@@ -73,13 +59,31 @@ export function createSystem(spec: SystemSpec): CreateAdminUIReturn {
                 crossOrigin="anonymous"
               />
             </Helmet>
-            <Global styles={styles(theme.global)} />
+            {!experimentalDisableGlobalStyles && (
+              <Global styles={styles(theme.global)} />
+            )}
             {children}
           </IconProvider>
         </SystemContext.Provider>
       </CacheProvider>
     )
   }
 
-  return [SystemProvider]
+  return [SystemProvider, emotion]
+}
+
+export interface CreateSystemOptions {
+  /** Kebab-case key */
+  key: string
+  /** Custom theme */
+  experimentalTheme?: any
+  /** Options of the created emotion instance */
+  experimentalEmotionOptions?: Omit<CreateEmotionOptions, 'key'>
+  /** Disable global styles */
+  experimentalDisableGlobalStyles?: boolean
```

Comment:
**Suggestion:** I would try to change the name to something like `isExperimentalGlobalStylesDisabled` or `experimentalDisabledGlobalStyles` to be easily identified as a boolean:

```suggestion
  experimentalDisabledGlobalStyles?: boolean
```
or
```suggestion
   isExperimentalGlobalStylesDisabled?: boolean?: boolean
```

Reply:
I would go with the first one

Reply:
I mean, I do see disableGlobalStyles as a boolean 🤔

Reply:
I think it is more related to an action (function) than a boolean for me.

Reply:
Makes sense

Reply:
Done